modules/nw-about-static-egress-ips.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * networking/enabling-static-egress-ips.adoc
+
+[id="about-{context}"]
+= About static IPs for external project traffic
```

Comment:
Most of the stuff in this section applies to both manual and fully-automatic egress IPs. It's an introduction to the whole concept of egress IPs, not just to manually-assigned egress IPs